table_test.go

```diff
@@ -43,11 +45,15 @@ func basicTable(t *testing.T, granuleSize int) *Table {
 		dynparquet.NewSampleSchema(),
 	)
 
+	bucket, err := filesystem.NewBucket(".")
+	require.NoError(t, err)
+
 	c := New(
 		nil,
 		granuleSize,
 		512*1024*1024,
-	)
+	).WithStorageBucket(bucket)
```

Comment:
I noticed that if I remove this, I start getting a test failure.

```
--- FAIL: Test_Table_InsertCancellation (0.06s)
    --- FAIL: Test_Table_InsertCancellation/1024 (0.06s)
        table_test.go:1115:
            	Error Trace:	table_test.go:1115
            	Error:      	"400" is not less than "400"
            	Test:       	Test_Table_InsertCancellation/1024
FAIL
exit status 1
FAIL	github.com/polarsignals/arcticdb	4.775s
```

Reply:
Very strange, I just run the following command (with and without the bucket) and it works.

```bash
go test -v -run Test_Table_InsertCancellation -count 100
```